index.html

```diff
@@ -104,7 +100,7 @@ <h2>Tonic</h2>
                             <li>javascript</li>
                         </ul>
                         <button class="card-button" type="submit">See Project</button>
-              </div>
+                    </div>
             </div>
 
             <div class="card two">  
```

Comment:
- [ ] I would kindly suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> Figma Template
![Cardsalign](https://user-images.githubusercontent.com/71140133/206710526-3c54544c-a827-4b08-a405-4afeb6187d77.PNG)

> Your Design
![errorcard](https://user-images.githubusercontent.com/71140133/206710686-3fc2c5e6-9e0d-4a79-9415-5fc2df8ed84f.PNG)

- [ ] Also, adjust the `margin-left` on the `Let's connect` element and the `social icons`. I would suggest your wrap both elements in a `div` 

> your design
![heroalign](https://user-images.githubusercontent.com/71140133/206711099-94856c25-6442-46a4-93fe-6ec3fb18e6a5.PNG)

> Figma Design

![rightalignd](https://user-images.githubusercontent.com/71140133/206711329-b728cba2-6895-4442-8ee6-46aff317eedf.PNG)